docs/openapi/components/schemas/credentials/USDAPermitToImportCredential.yml

```
@@ -348,6 +357,8 @@ example: |-
         }
       },
       "intendedUse": "fruits and vegetables",
-      "approvalDate": "2021-02-16"
+      "approvalDate": "2021-02-16",
+      "ppqHeadquarterOfficial": "APHIS eFile Automatic Issued Permit Regulated by 7 CFR 319.",
+      "permitConditions": "This permit authorizes the importation of the listed articles, under the conditions specified below. A copy of the first page of this permit must accompany all shipments authorized under this permit. For additional questions concerning this permit . . ."
```

An ellipsis contains no spaces.
```suggestion
      "permitConditions": "This permit authorizes the importation of the listed articles, under the conditions specified below. A copy of the first page of this permit must accompany all shipments authorized under this permit. For additional questions concerning this permit..."
```